README.md

```diff
@@ -1,215 +1,127 @@
+## 3. User cases
+Doris not only provides high concurrent low latency point query performance, but also provides high throughput queries of ad-hoc analysis. 
 
-Generally speaking, Doris is the technology combination of Google Mesa and Apache Impala. Mesa is a highly scalable analytic data storage system that stores critical measurement data related to Google's Internet advertising business. Mesa is designed to satisfy complex and challenging set of users' and systems' requirements, including near real-time data ingestion and query ability, as well as high availability, reliability, fault tolerance, and scalability for large data and query volumes. Impala is a modern, open-source MPP SQL engine architected from the ground up for the Hadoop data processing environment. At present, by virtue of its superior performance and rich functionality, Impala has been comparable to many commercial MPP database query engine. Mesa can satisfy the needs of many of our storage requirements, however Mesa itself does not provide a SQL query engine; Impala is a very good MPP SQL query engine, but the lack of a perfect distributed storage engine. So in the end we chose the combination of these two technologies.
+Doris not only provides batch data loading, but also provides near real-time mini-batch data loading. 
 
-Learning from Mesa's data model, we developed a distributed storage engine. Unlike Mesa, this storage engine does not rely on any distributed file system. Then we deeply integrate this storage engine with Impala query engine. Query compiling, query execution coordination and catalog management of storage engine are integrated to be frontend daemon; query execution and data storage are integrated to be backend daemon. With this integration, we implemented a single, full-featured, high performance state the art of MPP database, as well as maintaining the simplicity.
+Doris also provides high availability, reliability, fault tolerance, and scalability. 
 
-## 2. System Overview
+The simplicity (of developing, deploying and using) and meeting many data serving requirements in single system are the main features of Doris.
 
-Doris' implementation consists of two daemons: frontend (FE) and backend (BE). The following figures gives the overview of architecture and usage.
+## 4. Compile and install
 
-![](./docs/resources/palo_architecture.jpg)
+Currently support Docker environment and Linux OS: 
+Docker（Linux/Windows/Mac), Ubuntu 16.04+ and CentOS 7.5+
```

Comment:
Symbol ‘+’ should be removed, we may not support higher version.

Reply:
We need to resolve llvm compile issue.